src/test/java/org/broadinstitute/hellbender/tools/exome/CallCNLoHAndSplitsIntegrationTest.java

```diff
@@ -0,0 +1,83 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.apache.commons.io.FilenameUtils;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.ExomeStandardArgumentDefinitions;
+import org.broadinstitute.hellbender.tools.exome.acsconversion.ACSModeledSegment;
+import org.broadinstitute.hellbender.tools.exome.acsconversion.ACSModeledSegmentUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class CallCNLoHAndSplitsIntegrationTest extends CommandLineProgramTest {
+    private static final String ACNV_TEST_SUB_DIR = publicTestDir + "org/broadinstitute/hellbender/tools/exome";
+    private static final String TEST_SUB_DIR = publicTestDir + "org/broadinstitute/hellbender/tools/exome/cnlohcaller";
+
+    // Typically, this next file would be the output of a tangent normalization run.
+    private static final File COVERAGES_FILE = new File(ACNV_TEST_SUB_DIR, "coverages-for-allelic-integration.tsv");
+    private static final File TUMOR_ALLELIC_COUNTS_FILE = new File(ACNV_TEST_SUB_DIR, "snps-full.tsv");
+    private static final File SEGMENT_FILE = new File(TEST_SUB_DIR, "cell_line_small-sim-final.seg");
+
+    @Test
+    public void testBasicRun() {
+        final File outputDir = createTempDir("cnLoH_OutputDir_");
+        final List<String> arguments = new ArrayList<>();
+        arguments.add("-" + ExomeStandardArgumentDefinitions.TUMOR_ALLELIC_COUNTS_FILE_SHORT_NAME);
+        arguments.add(TUMOR_ALLELIC_COUNTS_FILE.toString());
+        arguments.add("-" + ExomeStandardArgumentDefinitions.TANGENT_NORMALIZED_COUNTS_FILE_SHORT_NAME);
+        arguments.add(COVERAGES_FILE.toString());
+        arguments.add("-" + ExomeStandardArgumentDefinitions.SEGMENT_FILE_SHORT_NAME);
+        arguments.add(SEGMENT_FILE.toString());
+        arguments.add("--verbosity");
+        arguments.add("INFO");
+        arguments.add("-" + CallCNLoHAndSplits.OUTPUT_DIR_SHORT_NAME);
+        arguments.add(outputDir.getAbsolutePath());
+        arguments.add("-" + CallCNLoHAndSplits.NUM_ITERATIONS_SHORT_NAME);
+        arguments.add("3");
+
+        runCommandLine(arguments);
+
+        final String gatkCnvFilename = outputDir.getAbsolutePath() + "/" +
+                FilenameUtils.removeExtension(SEGMENT_FILE.getAbsoluteFile().getName()) + "." +
```

Comment:
extract a private method for the `removeExtension . . .` pattern used several times in this test.